Cargo.toml

```diff
@@ -31,4 +31,4 @@ tokio = { version = "1.39.2", default-features = false, features = ["rt", "io-ut
 [features]
 default = ["egui"]
 cli = ["clap", "crossterm"]
-egui = ["rfd", "eframe", "egui-notify", "notify-rust"]
+egui = ["clap", "rfd", "eframe", "egui-notify", "notify-rust"]
```

Comment:
Instead of adding clap here, you could remove it from the cli feature and remove the `optional = true` on clap above.

Reply:
done